extractor/src/main/java/org/schabi/newpipe/extractor/services/media_ccc/extractors/MediaCCCStreamExtractor.java

```diff
@@ -215,11 +214,6 @@ public StreamType getStreamType() {
         return StreamType.VIDEO_STREAM;
     }
 
-    @Override
-    public StreamInfoItem getNextStream() {
-        return null;
-    }
-
     @Override
     public StreamInfoItemsCollector getRelatedStreams() {
```

Comment:
Now that `getNextStream()` is removed, we should add a JDoc here to say that the first related stream might be what was previously handled as "next stream".